homeassistant/components/opentherm_gw/button.py

```diff
@@ -30,6 +35,12 @@ class OpenThermButtonEntityDescription(
 
 
 BUTTON_DESCRIPTIONS: tuple[OpenThermButtonEntityDescription, ...] = (
+    OpenThermButtonEntityDescription(
```

Comment:
Could you add a test similar to `test_restart_button`? https://github.com/home-assistant/core/blob/bf9788b9c4724b46a0289342d6122477df2d883e/tests/components/opentherm_gw/test_button.py#L19

Reply:
Added and rebased on latest dev. Thanks for the input :+1: